apps/web/components/eventtype/RecurringEventController.tsx

```diff
@@ -32,11 +33,7 @@ export default function RecurringEventController({
       value: value.toString(),
     }));
 
-  const { shouldLockDisableProps } = useLockedFieldsManager(
-    eventType,
-    t("locked_fields_admin_description"),
-    t("locked_fields_member_description")
-  );
+  const { shouldLockDisableProps } = useLockedFieldsManager(eventType, formMethods, t);
```

Comment:
We now need form methods in order to set discretely the different fields in the event type metadata